docs/concepts/who.qmd

```diff
@@ -14,15 +14,18 @@ contribute to Ibis.
 
 ## Other companies
 
-Ibis is used by many other companies, with various tools built on top of it.
-These include:
+Ibis is contributed to used by many other companies. Various tools built on
+top of it. Some publicly available include:
```

Comment:
Hm, this looks grammatically incorrect and also now has a sentence fragment in it.

What change were you going for here?

Reply:
forgot an "and" 

lemme reword